### ISSUE TITLE
Screenshots in README

### ISSUE THREAD

**Issue by resynth1943** [Rafostar/clapper#5]
If I'm understanding correctly (I've been watching this repository), this media player can act as a standalone player. Would you like me to grab some screenshots of it running uhh... Big Buck Bunny (since that seems to be the favourite demo video) and add them to the README?

**Comment by Rafostar** [comment]
Hold on. The player interface is only **half made** right now. I will add some screenshots once GUI is finished.

**Comment by resynth1943** [comment]
Oh nice, can't wait for this thing to get stable :D Maybe it could be a good contender to VLC 😂

**Comment by Rafostar** [comment]
@resynth1943 

It took a little longer than expected, cause along the way I had to do a port of GStreamer to GTK4 and then my player too. The player is far from finished, but its already taking advantage of superb GTK4 + GStreamer GL rendering capabilities (plays videos fine).

> Maybe it could be a good contender to VLC

Right now when combined with the latest version of GTK4 + GStreamer 1.18 the average CPU usage when playing H.264 1080p videos on my 8 years old AMD APU is about 2% :smile:. VLC could never achieve such low CPU usage for me.

Today I took two pictures showing what I already have in both windowed mode and fullscreen mode (we have an adaptive UI). Now its about time for a little show-off. Introducing `Clapper` - the **first** media player build in GTK4.

Screenshots in Readme. Hope you like. Cheers!
